test/e2e/storage/testsuites/provisioning.go

```diff
@@ -694,7 +694,8 @@ func StopPod(c clientset.Interface, pod *v1.Pod) {
 		framework.Logf("Pod %s has the following logs: %s", pod.Name, body)
 	}
 	e2epod.DeletePodOrFail(c, pod.Namespace, pod.Name)
-	e2epod.WaitForPodNoLongerRunningInNamespace(c, pod.Name, pod.Namespace)
+	err = e2epod.WaitForPodNotFoundInNamespace(c, pod.Name, pod.Namespace, framework.PodDeleteTimeout)
```

Comment:
Most of our tests use `e2epod.DeletePodWithWait`

Reply:
I've switched to that.